src/steps.tsx

```diff
@@ -7,22 +8,17 @@ import Results from './components/results-bubble'
 const intro: Step[] = [
   {
     id: 'intro1',
-    message: 'Hi there!',
+    message: i18n.t('steps.intro1'),
```

Comment:
can we just set the keys here and add a transform? feels like less boilerplate that way :)

Reply:
we can even derive this from the `id` of the step (which is guaranteed unique), so no keys needed

Reply:
Only thing is it'll need to deep to handle labels, etc. Agree that it's ugly, though.